caffe2/python/operator_test/conv_test.py

```diff
@@ -19,6 +19,9 @@
 import unittest
 import os
 
+def _run_in_hip(gc, dc):
```

Comment:
as this appears multiple times in the codebase now, could you put it in a test_util file? (say hip_test_util.py)